src/initialize.js

```diff
@@ -130,13 +139,32 @@ export async function auth(requireUser, hydrateUser) {
     hydrateAuthenticatedUser();
   }
 }
+
+/**
+ * Set or overrides configuration via an env.config.js file in the consuming application.
+ * This env.config.js is loaded at runtime and must export one of two things:
+ *
+ * - An object which will be merged into the application config via `mergeConfig`.
+ * - A function which returns an object which will be merged into the application config via
+ * `mergeConfig`.  This function can return a promise.
+ */
+async function jsFileConfig() {
+  let config = {};
+  if (typeof envConfig === 'function') {
+    config = await envConfig();
```

Comment:
Hm, getting coverage on all these lines is challenging.  I'm not sure how I'd change the contents of the env.config file on a per-test basis. 🤔

Reply:
Same here, unfortunately. Just tried to play around with Jest mocks after pulling down the branch. I'm also not having much luck trying to mock the `env.config` import on a per-test basis... I was able to mock the import to always return a function, but then we don't get coverage on the non-function case. I was also able to mock it to return a custom object but then I couldn't find a way to change that mock object to a function in a second test.

Reply:
I figured it out.  I put each test in its own file and `jest.mock` for `env.config.js` with different implementations worked like a charm. :)  It also let me remove the real env.config.js file from the repo!  

I found some instructions online for doing it all in one file, but it didn't work for me, so I figure this will be decent for now.